src/components/beneficiary/EditForm.tsx

```diff
@@ -0,0 +1,211 @@
+        </Typography>
+        <Typography variant="body1" sx={{ textAlign: 'center', padding: '8px', mb: 2 }}>
+          {t('beneficiary:forms:info')}
+        </Typography>
+        <Grid container spacing={3}>
+          <Grid item xs={6}>
+            <FormControl fullWidth variant="outlined" size="small">
+              <InputLabel>{t('beneficiary:grid:type')}</InputLabel>
+              <Select
+                fullWidth
+                name="type"
+                disabled={Boolean(id)}
+                defaultValue={initialValues.type}
+                label={t('beneficiary:grid:type')}>
+                <MenuItem value={undefined} disabled>
+                  <em>None</em>
+                </MenuItem>
+                {Object.values(BeneficiaryType)?.map((type) => {
+                  return (
+                    <MenuItem key={type} value={type}>
+                      {t('beneficiary:grid:' + type)}
+                    </MenuItem>
+                  )
+                })}
+              </Select>
+            </FormControl>
+          </Grid>
+          <Grid item xs={6}>
+            <OrganizerRelationSelect
+              name="organizerRelation"
+              label="beneficiary:grid:organizerRelation"
+            />
+          </Grid>
+          {initialValues.type === BeneficiaryType.individual ? (
+            <>
+              <Grid item xs={12}>
+                <Typography paddingLeft={'inherit'} marginBottom={2}>
+                  {t('beneficiary:forms.labels.person-select')}{' '}
+                  <Link href={routes.admin.person.create}>
+                    {t('beneficiary:forms.labels.create-new')}
+                  </Link>
+                </Typography>
+                <PersonSelect name="personId" label={t('beneficiary:forms.labels.person-label')} />
+              </Grid>
+              <Grid item xs={6}>
+                <CountrySelect name="countryCode" />
+              </Grid>
+              <Grid item xs={6}>
+                <CitySelect
+                  disabled={initialValues.type === BeneficiaryType.company}
+                  name="cityId"
+                />
+              </Grid>
+            </>
+          ) : (
+            <Grid item xs={12}>
+              <Typography paddingLeft={'inherit'} marginBottom={2}>
+                {t('beneficiary:forms.labels.company-select')}{' '}
+                <Link href={routes.admin.company.create}>
+                  {t('beneficiary:forms.labels.create-new')}
+                </Link>
+              </Typography>
+              <CompanySelect name="companyId" label={t('beneficiary:forms.labels.company-label')} />
+            </Grid>
+          )}
+          <Grid item xs={12}>
+            <FormTextField
+              type="text"
+              name="description"
+              autoComplete="target-amount"
+              label={t('beneficiary:grid:description')}
+              multiline
+              rows={1.5}
+            />
```

Comment:
the autoComplete here seems not correct, please remove as it doesn't make sense for description field
also lets have rows=2